package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "weather-app-backend",
```

Comment:
can you add the following settings as well: 

- tslint settings 
- husky settings 
- code coverage settings
- prettier settings

if the husky fails, we don't want to allow github commit.

Reply:
I installed tslint, husky and prettier as devdependencies and setup .json files inserting some basic rules.
I ensured that when running npm test it will run with code coverage and the table will be displayed.